### PR TITLE
Icmslst 533 docker compose 2

### PR DESCRIPTION
**Pull request by trbateman**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,113 +2,111 @@
...
+      - DJANGO_SETTINGS_MODULE
+      - ELASTIC_APM_SECRET_TOKEN
+      - ELASTIC_APM_ENVIRONMENT
+      - ELASTIC_APM_URL
+      - DISABLE_DEBUG_TOOLBAR
+      - CLAM_AV_USERNAME
+      - CLAM_AV_PASSWORD
+      - CLAM_AV_URL
+    ports:
+      - "${ICMS_WEB_PORT}:${ICMS_WEB_PORT}"
```
Comment (kevinetienne) [inline docker-compose.yml:46]: Can you set a default value for `ICMS_WEB_PORT`.

```yam
    ports:
            - "${ICMS_WEB_PORT:-8080}:${ICMS_WEB_PORT:-8080}"
```

Comment (kevinetienne) [reply docker-compose.yml:46]: By setting a default port, we get flexibility and avoid breaking current setup.

**Review by oskusalerma — CHANGES_REQUESTED** [review]
i can't accept a full reformat of the file for no reason, no idea what's actually changed

File: docker-compose.yml (hunk 1 above)
Comment (trbateman) [reply docker-compose.yml]: Genius!

**Pull request closed without merging**